File: BEE 1149.py
list_num = list(map(int, input().split(" ")))
a = list_num[0]
n = 0
for i in range(1, len(list_num)):
    k = list_num[i]
    if (k > 0):
        n = k
        break

sum = 0
for i in range(n):
    sum += i + a

print(sum)    

File: BEE 1149.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var a = parseInt(line.shift());
var n = parseInt(line.shift());

while(n <= 0){ n = parseInt(line.shift());}

var sum = 0;
for (var i = 0; i < n; i++){
    sum += i + a;
}
console.log(sum);